README.md

```diff
@@ -1,2 +1,8 @@
 # lambda-megaphone-validator
 An AWS lambda function that compares the file sizes of recently uploaded audio files with the original files
+
```

Comment:
I'd add a note here about what environment variables are required here too, even though you have a .env.example

Reply:
@nakedsushi, Thank you for the suggestion! I went ahead and updated here as well.